README.md

```diff
@@ -558,6 +558,8 @@ const uiSchema = {
 };
 ```
 
+In addition it is valid to mark an array as "required".  This means the user must select at least one item to add to the array.
```

Comment:
The SelectWidget is an interesting case.  In truth I don't use it much because I've written my own custom widget based on react-select which better suited my needs for a nullable and multiselect-able drop down widget.  (Happy to share if anyone wants it, but I thought since it was an extra dependency it was better off outside the core project.)

Although it flies in the face of this change, I would tend to opt the other direction for the SelectWidget ... if only because the programmer has explicitly set the valid return values so should be expecting it, so perhaps we don't need to try and interpret their intent.